src/courtroom.cpp

```diff
@@ -4063,10 +4092,10 @@ void Courtroom::on_music_list_double_clicked(QTreeWidgetItem *p_item,
 {
   if (is_muted)
     return;
-
+  if (p_item->parent() == nullptr) // i.e. we've clicked a category
+    return;
```

Comment:
Wait is this a regression of double-clicking on music category stopping the music functionality? Shouldn't we make a settings option for this? I expect lots of people who are used to the old inferior style to complain they have no option to bring back old "ux is my passion" way.

Reply:
Over my dead body

Reply:
YES, DOWN WITH THE AWFUL UX